### PR TITLE
Re-enable wikimedia rule

### PR DESCRIPTION
**Pull request by jsha**
Add test URLs, remove some hosts that no longer exist.

cc @cooperq for review.

### CODE REVIEW

File: src/chrome/content/rules/Wikimedia.xml [hunk 1]
```diff
@@ -98,35 +129,32 @@ Fetch error: http://icinga.wmflabs.org/ => https://icinga.wmflabs.org/: (6, 'Cou
...
+	<rule from="^http://(?:download|sitemap)\.wikimedia\.org/"
 		to="https://dumps.wikimedia.org/" />
+		<test url="http://download.wikimedia.org/" />
+		<test url="http://sitemap.wikimedia.org/" />
 
 	<rule from="^http://links\.email\.donate\.wikimedia\.org/"
 		to="https://www.links.mkt41.net/" />
+		<test url="http://links.email.donate.wikimedia.org/" />
 
-	<rule from="^http://(?:labs-ns[01]|virt0)\.wikimedia\.org/"
```
Comment (reedy) [inline src/chrome/content/rules/Wikimedia.xml]: virt0 doesn't resolve in DNS, but both labs-ns1 and labs-ns0 do resolve

File: src/chrome/content/rules/Wikimedia.xml [hunk 2]
```diff
@@ -98,35 +129,32 @@ Fetch error: http://icinga.wmflabs.org/ => https://icinga.wmflabs.org/: (6, 'Cou
...
+		<test url="http://arbcom.en.wikipedia.org/" />
+		<test url="http://arbcom.fi.wikipedia.org/" />
+		<test url="http://arbcom.nl.wikipedia.org/" />
 
 	<rule from="^http://download\.wikipedia\.org/"
 		to="https://dumps.wikimedia.org/" />
+		<test url="http://download.wikipedia.org/" />
 
-	<rule from="^http://wg\.en\.wikipedia\.org/"
-		to="https://wg-en.wikipedia.org/" />
```
Comment (reedy) [inline src/chrome/content/rules/Wikimedia.xml]: This is an old url (that still resolves) but would've borked due to the wrong cert if you visit https://wg.en.wikipedia.org

File: src/chrome/content/rules/Wikimedia.xml [hunk 3]
```diff
@@ -70,13 +77,37 @@ Fetch error: http://icinga.wmflabs.org/ => https://icinga.wmflabs.org/: (6, 'Cou
...
+	<test url="http://ru.wikivoyage.org" />
+	<test url="http://ru.wiktionary.org" />
+
+	<test url="http://www.wikidata.org" />
+	<test url="http://www.wikimedia.org" />
+	<test url="http://www.wikimediafoundation.org" />
 
 	<!-- Wikimedia Tool Labs -->
 	<target host="tools.wmflabs.org" />
-	<target host="icinga.wmflabs.org" />
```
Comment (reedy) [inline src/chrome/content/rules/Wikimedia.xml]: Still resolves?

Comment (reedy) [inline src/chrome/content/rules/Wikimedia.xml]: C:\Users\Reedy>nslookup icinga.wmflabs.org
Server:  ACA80301.ipt.aol.com
Address:  172.168.3.1

Non-authoritative answer:
Name:    icinga.wmflabs.org
Address:  208.80.155.156


C:\Users\Reedy>

**Comment by cooperq** [comment]
Looks fine to me, modulo @reedy's comments. Merge when you are ready.

File: src/chrome/content/rules/Wikimedia.xml (hunk 1 above)
Comment (jsha) [inline src/chrome/content/rules/Wikimedia.xml]: Yes, but labs-ns1 times out on `curl labs-ns1.wikimedia.org`

File: src/chrome/content/rules/Wikimedia.xml (hunk 3 above)
Comment (jsha) [inline src/chrome/content/rules/Wikimedia.xml]: Hm, I think this one timed out or errored when I tried it, but now seems fine. I'll add it back in.

**Pull request merged**